package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "facebook-pixel-for-wordpress",
+  "version": "1.0.0",
+  "description": "Grow your business with Facebook for WordPress! This plugin will install a Facebook Pixel for your page so you can capture the actions people take when they interact with your page, such as Lead, ViewContent, AddToCart, InitiateCheckout and Purchase events. Version 2.0.0 also includes support for the Conversions API, which lets you send events directly from your page’s server so you can capture a more of these events when they happen. This can help you better understand your customer’s journey from the moment they show interest in your business to the moment they complete a conversion. You can use this information to create ad campaigns that are relevant to your audience. [Learn More](https://www.facebook.com/business/learn/facebook-ads-pixel)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
license should be: GPL-2.0

Reply:
Thank you for the notice!

I have just pushed an update for this: https://github.com/facebookincubator/Facebook-Pixel-for-Wordpress/pull/54/commits/1a2d5e48f7a798a47a652e48c8281419c6cf6a4a.